assessments/NWEA_Map/earthmover.yaml

```diff
@@ -168,11 +183,26 @@ destinations:
     template: ${BUNDLE_DIR}/templates/assessmentReportingMethodDescriptors.jsont
     extension: jsonl
     linearize: True
+  assessmentCategoryDescriptors:
```

Comment:
So my hesitation here is that now every implementation will attempt to send these descriptors and will in a lot of cases just repeatedly skip. Part of me thinks project composition should be used for this use case?

Reply:
I left this here but it's true of the grades/subjects too. It would be inconsistent for other assessments as well